src/PayU/Alu/Request.php

```diff
@@ -203,6 +210,11 @@ private function transformObject2Array()
             );
         }
 
+        if (isset($this->fx)) {
```

Comment:
`fx !== null` or `fx instanceof FX` would be more clear.

Reply:
Done!